CafeRecords/Pages/Index.cshtml

```diff
@@ -1,32 +1,59 @@
 ﻿@page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home page";
-    CafeRecord.Cafe[] allCafes = (CafeRecord.Cafe[])ViewData["allCafes"];
-    OwnerRecord.Owner[] allOwners = (OwnerRecord.Owner[])ViewData["allOwners"];
+    
     List<OwnerRecord.Owner> cafeOwners = (List<OwnerRecord.Owner>)ViewData["cafeOwners"];
 }
+    <html>
+    <head>
+        <style>
+            body {
+                background-image: url("https://desktopwalls.net/wp-content/uploads/2014/07/Chicago%20Colorful%20Skyline%20View%20Desktop%20Wallpaper.jpg");
+                background-repeat: no-repeat;
+                background-size:cover;
+            }
+        </style>
+    </head>
+    </html>
+
+        <div class="text-center">
+            <h1 class="display-4" color><strong>Cafe Records</strong></h1>
+            <br />
+
+            <style>
+                table {
+                    border-collapse: collapse;
```

Comment:
added a table to improve the look and feel.

Reply:
Ditto comment above... great idea... move to linked stylesheet will make it even better.